packages/client/src/components/app/forms/MultiFieldSelect.svelte

```diff
@@ -28,6 +29,18 @@
     valueColumn,
     customOptions
   )
+
+  const expand = values => {
+    if (!values) {
+      return []
+    }
+
+    if (Array.isArray(values)) {
+      return values
+    }
+
+    return values.split(",")
```

Comment:
To make this setting a little more user friendly, we could trim the values in case someone enters a default value with spaces after the commas.
```js
return values.split(",").map(value => value.trim())
```